### PR TITLE
clean up tools lathe recipes

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
clean up hydro and tools files, and changed some times:
- a single cable now takes 0.1s to print instead of 2, literally no reason it was that long with 1 cable in mind
- rcd ammo standardized to 2 seconds from 2.4 nobodys gonna lose sleep over the 400ms saved

requires #31515

## Why / Balance
-diff

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- tweak: Printing cables in an autolathe is now 20 times faster.

### CODE REVIEW

File: Resources/Prototypes/Recipes/Lathes/botany.yml [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
+  id: BaseHydroToolRecipe
   materials:
     Steel: 200
     Plastic: 100
 
+# Recipes
+
 - type: latheRecipe
+  parent: BaseHydroToolRecipe
+  id: MiniHoe
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Lathes/botany.yml:15]:
```suggestion
  id: HydroponicsToolMiniHoe
```

File: Resources/Prototypes/Recipes/Lathes/botany.yml [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
+  result: HydroponicsToolMiniHoe
+
+- type: latheRecipe
+  parent: BaseHydroToolRecipe
   id: HydroponicsToolScythe
-  result: HydroponicsToolScythe
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Lathes/botany.yml:11]: keep result

File: Resources/Prototypes/Recipes/Lathes/botany.yml [hunk 3]
```diff
@@ -1,39 +1,37 @@
...
-  completetime: 2
   materials:
     Steel: 300
     Plastic: 200
 
 - type: latheRecipe
+  parent: BaseHydroToolRecipe
   id: HydroponicsToolHatchet
-  result: HydroponicsToolHatchet
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Lathes/botany.yml:19]: literally 1984

File: Resources/Prototypes/Recipes/Lathes/cooking.yml [hunk 4]
```diff
@@ -22,46 +22,39 @@
...
+- type: latheRecipe
+  id: CustomDrinkJug
+  result: CustomDrinkJug
+  completetime: 2
+  materials:
     Plastic: 200
-    
+
 - type: latheRecipe
+  parent: DrinkMug
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Lathes/cooking.yml:55]: don't inherit the plate from the mug

File: Resources/Prototypes/Recipes/Lathes/cooking.yml [hunk 5]
```diff
@@ -99,8 +90,6 @@
     Steel: 100
 
 - type: latheRecipe
+  parent: FoodPlateTin
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Lathes/cooking.yml:93]: you save 2 lines with this but the parenting makes no sense

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Comment by slarticodefast** [comment]
Are you coming back to this?

**Comment by deltanedas** [comment]
yes

**Comment by slarticodefast** [comment]
Looks good to me, thanks for the cleanup.

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**